src/codelabs/implicit-animations.md

```diff
@@ -244,22 +244,21 @@ Click the **Run** button to run the example:
 
 ### Animate color, borderRadius, and margin with AnimatedContainer
 
-  This section contains a list of steps you can use to add an
-  implicit animation to the [shape-shifting starter code][].
-  After the steps, you can also run the
-  [shape-shifting complete][] example with the changes already made.
+This section contains a list of steps you can use to add an
+implicit animation to the [shape-shifting starter code][].
+After the steps, you can also run the
+[shape-shifting complete][] example with the changes already made.
 
 In the [shape-shifting starter code][],
 each property in the `Container` widget (`color`,
 `borderRadius`, and `margin`)
 is assigned a value by an associated function (`randomColor()`,
 `randomBorderRadius()`, and `randomMargin()` respectively).
-By using an `AnimatedContainer` widget,
-you can refactor this code to do the following:
+To refactor this code, use an `AnimatedContainer` widget to:
 
-* Generate new values for `color`, `borderRadius`,
-  and `margin` whenever the user clicks the **Change** button.
-* Animate the transition to the new values for `color`,
+- Generate new values for `color`, `borderRadius`,
+  and `margin` whenever the user clicks **Change**.
+- Animate the transition to the new values for `color`,
```

Comment:
It's not that they use `AnimatedContainer` to generate new values, but rather it allows them to transition to the new values.